docs/xmlprofiles.rst

```diff
@@ -804,9 +804,6 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
 | |usewriliv|                    | Indicates to use the             | ``Boolean``             | :class:`true`         |
 |                                | WriterLiveliness protocol.       |                         |                       |
 +--------------------------------+----------------------------------+-------------------------+-----------------------+
-| ``<domainId>``                 | DomainId to be used by           | ``UInt32``              | 0                     |
```

Comment:
Include it in participant profiles attributes. You'll need to update the description of the Participant Profiles section, since right now it says everything is under `<rtps>`